ocsci/pytest_customization/ocscilib.py

```diff
@@ -103,3 +103,13 @@ def process_cluster_cli_params(config):
         cluster_path = os.path.join(cluster_dir_parent, cluster_name)
     ocsci_config.ENV_DATA['cluster_name'] = cluster_name
     ocsci_config.ENV_DATA['cluster_path'] = cluster_path
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """
+    Add Polarion ID property to test cases that are marked with one.
+    """
+    for item in items:
+        for marker in item.iter_markers(name="polarion_id"):
+            polarion_id = marker.args[0]
```

Comment:
Please add a skip when we dont have an ID

Reply:
Do you mean a situation when there is a "polarion_id" marker, but used without argument, like this?

```
@pytest.mark.polarion_id
def test_foo_bar(setup_something):
    ...
```

In such case, the pytest immediately fails with `IndexError: tuple index out of range` exception.
And it is true that we should handle this error better in some way. But I would rather not skip or ignore it without some warning.

Reply:
Yeah when the ID is missing we will hit the error you pasted above,  if we can handle this in unittest that will be great but if not we can handle here.

Reply:
> And it is true that we should handle this error better in some way. But I would rather not skip or ignore it without some warning.

I agree with this, we can raise a warning in that scenario so it doesn't go unnoticed.

Reply:
Done